### PR TITLE
PUBDEV-4350: Update R/Py docs about sample_rate_per_class

### PR DESCRIPTION
**Pull request by navdeep-G**
`sample_rate_per_class` does not currently have a clear definition. This PR clarifies any potential confusion (see JIRA: https://0xdata.atlassian.net/browse/PUBDEV-4350?filter=-1)

### CODE REVIEW

**Comment by navdeep-G** [comment]
@h2o-ops please test!

**Comment by navdeep-G** [comment]
@h2o-ops please test!

File: h2o-algos/src/main/java/hex/schemas/SharedTreeV3.java [hunk 1]
```diff
@@ -73,7 +73,7 @@
     @API(help = "Row sample rate per tree (from 0.0 to 1.0)", gridable = true)
     public double sample_rate;
 
-    @API(help = "Row sample rate per tree per class (from 0.0 to 1.0)", level = API.Level.expert, gridable = true)
+    @API(help = "A list/vector of the fraction of rows to sample for each tree, per class (from 0.0 to 1.0)", level = API.Level.expert, gridable = true)
```
Comment (arnocandel) [inline h2o-algos/src/main/java/hex/schemas/SharedTreeV3.java]: it's not fraction of rows, but fraction of rows belonging to each class. So I'd say this:

A list of row sample rates per class (relative fraction for each class, from 0.0 to 1.0), for each tree.

**Comment by navdeep-G** [comment]
@arnocandel Updated based on feedback

**Comment by navdeep-G** [comment]
@h2o-ops please test!

**Pull request merged**